src/app/plugins/reciperecommender.plugin.ts

```diff
@@ -0,0 +1,53 @@
+import axios from 'axios';
+import { MessageEmbed } from 'discord.js';
+import Constants from '../../common/constants';
+import { Plugin } from '../../common/plugin';
+import { ChannelType, IContainer, IMessage } from '../../common/types';
+
+export default class RecipeRecommenderPlugin extends Plugin {
+  public commandName: string = 'reciperecommender';
+  public name: string = 'Recipe Recommender';
+  public description: string =
+  'Recommends a random recipe from http://whatthefuckshouldimakefordinner.com/ or its vegetarian counterpart, given the optional argument.';
+  public usage: string = 'recrec <veg>?';
+  public permission: ChannelType = ChannelType.Public;
+  public pluginAlias: string[] = ['recrec', 'rr', 'reciperecommender'];
+  public pluginChannelName = Constants.Channels.Public.Food;
+
+  private static readonly _API_URL = 'http://whatthefuckshouldimakefordinner.com/';
+  private static readonly _RECIPE_NAME_REGEX = /\>(.*?)\</g;
+  private static readonly _IMAGE_PREVIEW_REGEX = /<meta.*property="og:image".*content="(.*)".*\/>/;
+  private static readonly _URL_REGEX = /(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&/=]*)/;
+  private static readonly _AUTHOR_IMAGE_URL = 'https://i.imgur.com/B2vwvFL.png';
+
+  constructor(public container: IContainer) {
+    super();
+  }
+
+  public async execute(message: IMessage, args: string[]) {
+    const wantsVeg = args[0] === 'veg';
```

Comment:
Does this not create an error when an argument is no given?

Also, should use `toLowerCase()`

Reply:
Doesnt give an error, will make the lowercase change